thumbor/handlers/__init__.py

```diff
@@ -638,8 +638,8 @@ def _fetch(self, url):
     def get_blacklist_contents(self):
         filename = 'blacklist.txt'
 
-        exists = yield gen.maybe_future(self.context.modules.storage.exists(filename))
-        if exists:
+        exist = yield self.context.modules.storage.exists(filename)
+        if exist:
```

Comment:
Try not to rename variables and leave only changes related to the coroutine rewrite. Also, I believe `exists` to be a better suited name for that variable.